pandera/strategies.py

```diff
@@ -384,9 +422,8 @@ def eq_strategy(
     :returns: ``hypothesis`` strategy
     """
     # override strategy preceding this one and generate value of the same type
-    if strategy is None:
-        strategy = pandas_dtype_strategy(pandera_dtype)
-    return st.just(value).map(to_numpy_dtype(pandera_dtype).type)
+    # pylint: disable=unused-argument
+    return pandas_dtype_strategy(pandera_dtype, st.just(value))
```

Comment:
@cosmicBboy `strategy` had already no effect prior to my commit. I guess the purpose of this argument is compatibility with strategy chains?

Reply:
yep!